### ISSUE TITLE
Rubocop CI - failing on new 3.1 code (BlockLength metric possible)

### ISSUE THREAD

**Issue by luke-hill** [rubocop/rubocop#11127]
--------

## Expected behavior

Rubocop not to error

## Actual behavior

Rubocop threw some errors in CI - For BlockLength

```
An error occurred while Layout/BlockAlignment cop was inspecting /home/runner/work/site_prism/site_prism/spec/site_prism/deprecator_spec.rb:3:0.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/BlockAlignment cop was inspecting /home/runner/work/site_prism/site_prism/spec/site_prism/addressable_url_matcher_spec.rb:3:0.
To see the complete backtrace run rubocop -d.
```

## Steps to reproduce the problem

Run the CI in this OSS repo

## RuboCop version

```
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.21.0 (using Parser 3.1.2.1, rubocop-ast 1.23.0, running on ruby 3.1.2 x[86](https://github.com/site-prism/site_prism/actions/runs/3338598716/jobs/5526510045#step:6:87)_64-linux)
Inspecting 74 files
```

**Comment by koic** [comment]
I tried running it in the https://github.com/site-prism/site_prism repository. Those errors in RuboCop 1.21 are resolved in RuboCop 1.22 and higher. Can you upgrade RuboCop to the latest version? Thank you.

**Comment by luke-hill** [comment]
Thanks for investigation. I'm quite behind I'll be honest. I **do need** to upgrade those and in some branches I am trying to.

Thanks again!
